Unity/Assets/YarnSpinner/Scripts/DialogueRunner.cs

```diff
@@ -353,14 +353,6 @@ void Start ()
             
         }
 
-        private void Update()
```

Comment:
This was forgotten testing thing that I forgot to remove before doing first commit.